<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>571</width>
     <height>351</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>General Settings</string>
    </attribute>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>561</width>
       <height>321</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="toolBarStyleLabel">
        <property name="text">
         <string>ToolBar Style</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="toolbarStyleComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>Icons Only</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Text Only</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Text Alongside Icons</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Text Under Icons</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>System Style</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="logLevelLabel">
        <property name="text">
         <string>Log Level</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="logLevelComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>ALL</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>INFO</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>WARN</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ERROR</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>FATAL</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>OFF</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="haproxyModeLabel">
        <property name="text">
         <string>Haproxy Mode</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="haproxyModeComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>roundrobin</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>leastconn</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>source</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="onlyOneInstanceLabel">
        <property name="text">
         <string>Allow Only one instance running</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="oneInstanceCheckBox">
        <property name="text">
         <string>Enable</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="checkPortAvailabilityLabel">
        <property name="text">
         <string>Check Port Availability</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QCheckBox" name="availabilityCheckBox">
        <property name="text">
         <string>Enable</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="enableNotificationLabel">
        <property name="text">
         <string>Enable Notification</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QCheckBox" name="enableNotificationCheckBox">
        <property name="text">
         <string>Enable</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="hideDockIconLabel">
        <property name="text">
         <string>Hide Dock Icon (macOS)</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QCheckBox" name="hideDockIconCheckBox">
        <property name="text">
         <string>Enable</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="startAtLoginLabel">
        <property name="text">
         <string>Start at login</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QCheckBox" name="startAtLoginCheckbox">
        <property name="text">
         <string>Enable</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="hideWindowsOnStartupLabel">
        <property name="text">
         <string>Hide windows on startup</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QCheckBox" name="hideCheckBox">
        <property name="text">
         <string>Enable</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="useNativeMenuBar">
        <property name="text">
         <string>Use native menubar</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QCheckBox" name="nativeMenuBarCheckBox">
        <property name="text">
         <string>Enable</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Theme</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="themeComboBox"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="widget">
    <attribute name="title">
     <string>Inbound Settings</string>
    </attribute>
    <widget class="QWidget" name="formLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>561</width>
       <height>321</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Enable Http Proxy</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="enableHttpProxyCheckBox">
        <property name="text">
         <string>Enable</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Enable IPV6 Support</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="enableIPV6SupportCheckBox">
        <property name="text">
         <string>Enable</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Share Over LAN</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="shareOverLANCheckBox">
        <property name="text">
         <string>Enable</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Sock5 Port</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="socks5PortLineEdit"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>Http Port</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="httpPortLineEdit"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>PAC Port</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="pacPortLineEdit"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>Haproxy Port</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="haproxyPortLineEdit"/>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>Haproxy Status Port</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="haproxyStatusPortLineEdit"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Outbond Settings</string>
    </attribute>
    <widget class="QWidget" name="formLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>561</width>
       <height>321</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_4">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_26">
        <property name="text">
         <string>Forward Proxy</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_27">
        <property name="text">
         <string>IP Address</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_30">
        <property name="text">
         <string>Port</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_28">
        <property name="text">
         <string>Authenticaton</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_31">
        <property name="text">
         <string>Username</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="forwardProxyUsername"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_32">
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="forwardProxyPassword"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_33">
        <property name="text">
         <string>Type</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="forwardProxyCheckBox">
        <property name="text">
         <string>Enable</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="forwardProxyTypeComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>http</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>socks5</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="forwardProxyIpAddressLineEdit"/>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="forwardProxyPortLineEdit"/>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="forwardProxyAuthenticationCheckBox">
        <property name="text">
         <string>Enable</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Subscribe Settings</string>
    </attribute>
    <widget class="QWidget" name="formLayoutWidget_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>561</width>
       <height>321</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_5">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="trojanBackendLabel_2">
        <property name="text">
         <string>GFWList Update Url</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="gfwlistUpdateUrlComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>gfwlist/gfwlist</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Loukky/gfwlist-by-loukky</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Local File</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="enableAPILabel_2">
        <property name="text">
         <string>Update User Agent</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="updateUserAgentLineEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="apiPortLabel_2">
        <property name="text">
         <string>Filter Keyword (,)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="filterKeywordLineEdit"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Maximum</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="maximumSubscribe"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Trojan Settings</string>
    </attribute>
    <widget class="QWidget" name="formLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>561</width>
       <height>321</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>TLS Fingerprint</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="tlsFingerprintComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>none</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>auto</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>firefox</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>chrome</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ios</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>randomized</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="enableRouterLabel">
        <property name="text">
         <string>Enable Router</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="enableRouterCheckBox">
        <property name="text">
         <string>Enable</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="enableAPILabel">
        <property name="text">
         <string>Enable API</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="enableAPICheckBox">
        <property name="text">
         <string>Enable</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="apiPortLabel">
        <property name="text">
         <string>API Port</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="apiPortLineEdit"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="certLabel">
        <property name="text">
         <string>Cert</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="certLineEdit"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="cipherLabel">
        <property name="text">
         <string>Cipher</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="cipherLineEdit"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="cipherTLS13Label">
        <property name="text">
         <string>Cipher TLS13</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="cipherTLS13LineEdit"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>417</x>
     <y>360</y>
     <width>164</width>
     <height>32</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>486</x>
     <y>375</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
